conf.yaml

```diff
@@ -782,6 +783,18 @@ contents:
                   -
                     repo:   apm-agent-python
                     path:   docs
+              -
+                title:      APM Ruby Agent
+                prefix:     python
+                current:    1.x
```

Comment:
Change to say `current: master`.

You can disregard my previous comments. I deleted them when I realized that you don't really need to override the label because you won't see a label when you have only one branch.